packages/annotation-converter/src/converter.ts

```diff
@@ -1541,6 +1542,40 @@ function convertTypeDefinition(converter: Converter, rawTypeDefinition: RawTypeD
     return convertedTypeDefinition;
 }
 
+function getReferences(rawMetadataReferences: Reference[]) {
+    if (rawMetadataReferences.length === 0) {
+        return VocabularyReferences;
+    }
+
+    // the provided references must be unique
```

Comment:
this check is probably a bit paranoid (O(n^2)) given that this was not checked at all before?

Reply:
Hmmm maybe not throwing but maybe we should keep it for the diagnostics, what do you think ?

Reply:
Makes sense. This no longer throws now